app/code/Magento/MediaGalleryUi/Test/Mftf/ActionGroup/AdminMediaGalleryEditAssetRemoveKeywordActionGroup.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminMediaGalleryEditAssetRemoveKeywordActionGroup">
+        <annotations>
+            <description>Remove Keywords on the Edit Details panel</description>
+        </annotations>
+        <arguments>
+            <argument name="selectedKeywords"/>
```

Comment:
I'd change the name and specify a type to make it more clear how to use this action group
```suggestion
            <argument name="keyword" type="string"/>
```